contributing.md

```diff
@@ -354,13 +100,14 @@ ansible-playbook --skip-tags tests::cleanup \
   `centos/systemd` is centos7.
 
     ```bash
-    podman run --name lsr-ci-c7 --rm --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
+    podman run --name lsr-ci-c7 --rm --privileged \
+    -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
     -d registry.centos.org/centos/systemd
     ```
 
-5. Use `podman unshare` first to run "podman mount" in root mode, use `-vi` to run
-  ansible as inventory in verbose mode, use `-c podman` to use collection plugins. Note,
-  the following tests are currently not working with podman:
+5. Use `podman unshare` first to run "podman mount" in root mode, use `-vi` to
+  run ansible as inventory in verbose mode, use `-c podman` to use collection
+  plugins. Note, the following tests are currently not working with podman:
```

Comment:
The patch looks good to me, but the tests changed a lot in the near past, I think the list below should be updated according to https://github.com/linux-system-roles/network/blob/main/.github/run_test.sh.